### PR TITLE
Docs: fixed some typos in config.yml.md.

### PR DESCRIPTION
**Pull request by AlexSkrypnyk**
- Fixed some typos.
- Added links to default.config.yml and a list of roles in the repo.
- Moved description about `default.config.yml` to the top of the page for visibility.

### CODE REVIEW

File: docs/configuration/ansible/config.yml.md [hunk 1]
```diff
@@ -4,20 +4,23 @@ This is the main method to customise the VM.
 
 In this file you can override any ansible variables provided by beetbox or by the galaxy roles.
 
+See the [beetbox.config.yml](https://github.com/beetboxvm/beetbox/blob/master/provisioning/ansible/config/default.config.yml) for a full list of variables.
```
Comment (AlexSkrypnyk) [inline docs/configuration/ansible/config.yml.md:7]: moved this to the top as it was completely lost among other text.

**Comment by thom8** [comment]
@alexdesignworks make sure your Circle CI build environment is set to `14.04`

image

**Comment by thom8** [comment]
@alexdesignworks -- https://circleci.com/docs/build-image-trusty/

> Please note that you need to trigger a build by pushing commits to GitHub or Bitbucket (instead of rebuilding) to apply the new setting.

Trying to convert tests to docker so this won't be a requirement

**Comment by AlexSkrypnyk** [comment]
@thom8 Please review and merge as build is now passed.

**Review by thom8 — APPROVED** [review]

**Pull request merged**